packages/kbn-guided-onboarding/src/components/landing_page/guide_card_footer.tsx

```diff
@@ -117,7 +123,7 @@ export const GuideCardFooter = ({ guides, useCase, activateGuide }: GuideCardFoo
         <EuiFlexItem grow={false}>
           <EuiButton
             // Used for FS tracking
-            data-test-subj={`onboarding--guideCard--continue--${useCase}`}
+            data-test-subj={`onboarding--guideCard--continue--${telemetryId}`}
```

Comment:
TODO: check with Os about changing telemetry ids

Reply:
This should be OK. AFAIK we are not tracking the continue button for 8.6. See https://github.com/elastic/kibana/issues/138564#issuecomment-1289861062.